pulid_client.py

```
@@ -0,0 +1,240 @@
+            "seed": arguments.get("seed"),
+            "id_weight": arguments.get("id_weight", 1.0),
+            "start_step": arguments.get("start_step", 0),
+            "true_cfg": arguments.get("true_cfg", 1.0),
+            "timestep_to_start_cfg": arguments.get("timestep_to_start_cfg", 1),
+            "max_sequence_length": int(arguments.get("max_sequence_length", "128"))
+        }
+
+        if opts["seed"] is None or opts["seed"] == -1:
+            opts["seed"] = torch.Generator(device="cpu").seed()
+
+        print(f"🎨 Generating image with PuLID...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)